testing/code/test_excinfo.py

```diff
@@ -1361,40 +1371,30 @@ def a(x):
     def b(x):
         return a(numpy_like())
 
-    try:
+    with pytest.raises(RecursionError) as excinfo:
```

Comment:
`RecursionError` is py35+ (or py36+) only; IIRC you need to use `RuntimeError` instead.

Reply:
dang, I always forget this 😭